specs/DotNetProjectFile.Analyzers.Specs/Design_specs.cs

```diff
@@ -30,7 +32,9 @@ public void Root_Readme_mentions_right_number_of_rules()
         var amount = int.Parse(match.Groups["amount"].Value);
 
         var all = AlleRules.Count();
```

Comment:
Dutch

Reply:
What do you mean?

Reply:
`AlleRules` -> `AllRules`, but it's not part of this PR otherwise, so it's fine, I just noticed it.

Reply:
I really did not spot this... 😕